.gitmodules

```diff
@@ -18,7 +18,7 @@
 	url = https://github.com/okdshin/unique_resource.git
 [submodule "vendor/boost"]
 	path = vendor/boost
-	url = https://github.com/mapbox/mapbox-gl-native-boost.git
+	url = https://github.com/kiibimees/mapbox-gl-native-boost.git
```

Comment:
We should probably ask for a proper MapLibre fork.

Reply:
Yes, I agree. It's discussed here https://github.com/maplibre/maplibre-gl-native/issues/124 but it doesn't progress, for some reason :)

Reply:
I would be so glad if any of the maintainers resolve https://github.com/maplibre/maplibre-gl-native/issues/124 and we could point this submodule to a more correct repository.